.gitignore

```diff
@@ -21,3 +21,4 @@ yarn.lock
 .nyc_output
 .benchmarks/
 *.csv
+.vscode
```

Comment:
Please add [these](https://dev.to/oleksiyrudenko/please-do-not-pollute-projects-gitignore-3h8e) to your own global gitignore